### PR TITLE
feat: added files for input component

### PR DESCRIPTION
**Pull request by Ranjeetk489**

### CODE REVIEW

File: components/Input/input.css [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+.pwd-flex {
+  display: flex;
+  flex-direction: column;
+  padding: 1em;
+}
+.msg {
+  margin-top: 10px;
+  font-weight: var(--font-weight-3);
+}
+.error-hl {
```
Comment (sanjitsarkar) [inline components/Input/input.css:39]: what does hl mean?
try to give meaningful name so that other can understand

**Review by sanjitsarkar — APPROVED** [review]

File: components/Input/input.css [hunk 2]
```diff
@@ -0,0 +1,50 @@
+.input {
+  padding: 12px;
```
Comment (DivyanshSareen) [inline components/Input/input.css:2]: can use rems

**Review by DivyanshSareen — APPROVED** [review]
good to merge

File: components/Input/input.css (hunk 1 above)
Comment (Ranjeetk489) [reply components/Input/input.css:39]: Highlight i think... I will try to be more meaningful and will update the existing code .. thanks

**Pull request merged**